src/Components/BestSellers/BestSellerCard.jsx

```diff
@@ -0,0 +1,19 @@
+export default function BestSellerCard({ card }) {
+    const { source, title } = card;
+    return (
+        <div className="card card-with-badge best-seller-card">
+            <div className="card-image">
+                <img src={source} alt="bestSeller" />
```

Comment:
you may give a dynamic alt too, so that it is not same always

Reply:
Will do it once the backend data is in place